src/promql.grammar

```diff
@@ -198,7 +198,7 @@ SubqueryExpr {
 }
 
 UnaryExpr {
-  !mul UnaryOp Expr
+  !mul UnaryOp~statement Expr
```

Comment:
I think you can replace `statement` with any word that is descriptive, like `signed` or something like that.

Reply:
ah yeah that works. I thought it was a keyword in lezer. I put `signed` didn't have a better idea so far